scripts/coverage.sh

```diff
@@ -32,12 +33,21 @@ generate_cover_data() {
 
     for pkg in "$@"; do
         f="$workdir/$(echo $pkg | tr / -).cover"
-        go test -v -covermode="$mode" -coverprofile="$f" "$pkg"
+        # go test -v -covermode="$mode" -coverprofile="$f" "$pkg"
+        # not using -v verbose flag, too much info to parse easily; output to file
+        go test -covermode="$mode" -coverprofile="$f" "$pkg" >> "$report"
```

Comment:
Nothing to address in this patch, but we probably need a separate CI job to run tests without coverage but printed to standard output, so that we can see the possible errors.